transforms/language/language_id/ray/Makefile

```diff
@@ -0,0 +1,51 @@
+# Define the root of the local git clone for the common rules to be able 
+# know where they are running from.
+REPOROOT=../../../..
+# Include a library of common .transform.* targets which most
+# transforms should be able to reuse.  However, feel free
+# to override/redefine the rules below. 
+include $(REPOROOT)/transforms/.make.transforms
+
+TRANSFORM_NAME=lang_id
```

Comment:
in general, it is better if the transform name  matches the name of the directory under the transforms directory (i.e. language_id in this case).   I would recommend changing the name of the directory from language_id to lang_id.